cognite/replicator/datapoints.py

```diff
@@ -75,18 +93,23 @@ def retrieve_insert(
                 client_dst.datapoints.insert(datapoints, external_id=ext_id)
             updated_timeseries_count += 1
             total_datapoints_copied += len(datapoints)
-        except CogniteAPIError:
+        except (CogniteAPIError, ValueError):

```

Comment:
That is a lot of code covered by one except clause.
Overall I think we need to refactor this a bit. Maybe we should merge it first, and then if it works, try to refactor it.

Reply:
This is slightly hacky on my part to catch a specific ValueError that I was obeserving, but I can avoid it in a nicer way.